pkg/k8s/labels.go

```diff
@@ -95,7 +95,7 @@ func isInjectedWithIstioSidecarProxy(scopedLog *logrus.Entry, pod *corev1.Pod) b
 }
 
 // GetPodLabels returns the labels of a pod
-func GetPodLabels(namespace, podName string) (map[string]string, error) {
+func GetPodLabels(namespace, podName string) (lbls map[string]string, retAnno map[string]string, retErr error) {
```

Comment:
This is no longer just getting pod labels, should it be renamed?

```suggestion
func GetPodMetadata(namespace, podName string) (lbls map[string]string, retAnno map[string]string, retErr error) {
```